packages/checkout-ui-extensions/src/components/Form/README.md

```diff
@@ -1,21 +1,9 @@
 # Form
 
-The `Form` component should be used to wrap one or more form controls.
-This component provides an &#34;implicit submit&#34; behavior, where buyers can
-submit the form from any input by pressing &#34;enter&#34; on their keyboards. This
-behavior is widely expected, and should be respected as often as possible.
-
-Unlike an HTML `form` element, this component does not support configuring
-the descendant fields to be submitted via HTTP automatically. Instead, you
-must provide an `onSubmit` callback that will perform the necessary HTTP
-requests in JavaScript.
+The `Form` component should be used to wrap one or more form controls. This component provides an &#34;implicit submit&#34; behavior, where buyers can submit the form from any input by pressing &#34;enter&#34; on their keyboards. This behavior is widely expected, and should be respected as often as possible.
```

Comment:
So strange that we use entities instead of the real characters :S

Reply:
It’s intentional because it was HTML, now markdown so this can be removed